libs/CMakeLists.txt

```diff
@@ -140,11 +137,6 @@ foreach(lib ${HPX_CANDIDATE_LIBS})
   if (HPX_${lib}_LIBRARY_ENABLED)
     set(HPX_LIBS ${HPX_LIBS} ${lib} CACHE INTERNAL "list of Enabled HPX modules" FORCE)
 
-    string(TOUPPER ${lib} uppercase_lib)
-
-    set(MODULE_ENABLED_LIB_DEFINES
-      "${MODULE_ENABLED_LIB_DEFINES}#define HPX_HAVE_LIB_${uppercase_lib}\n")
-
```

Comment:
I guess this was removed just because it's unused?

Reply:
Hmmm. This is missing from the module generator script. I'll fix that independently.

Reply:
See #4529